train.py

```diff
@@ -1,72 +1,86 @@
 import cv2
 import numpy
+import sys
 
-from lib.utils import get_image_paths, load_images, stack_images
-from lib.training_data import get_training_data
+from utils import get_image_paths, load_images, stack_images
+from training_data import get_training_data
 
-from lib.model import autoencoder_A
-from lib.model import autoencoder_B
-from lib.model import encoder, decoder_A, decoder_B
+from model import autoencoder_A
+from model import autoencoder_B
+from model import encoder, decoder_A, decoder_B
+from getopt import getopt
+
+
+try:
+    opts, args = getopt(sys.argv[1:], "a:b:", ["image-a=", "image-b="])
+except GetoptError as err:
+    print(err)
+    sys.exit(2)
+for option, value in opts:
+    if option in ("-a", "--image-a"):
+        imageAFolder = value
+    elif option in ("-b", "--image-b"):
+        imageBFolder = value
+    else:
+        assert False, "Unhandled option"
+
+print("Image A path is {}".format(imageAFolder))
+print("Image B path is {}".format(imageBFolder))
 
 try:
-    encoder.load_weights('models/encoder.h5')
-    decoder_A.load_weights('models/decoder_A.h5')
-    decoder_B.load_weights('models/decoder_B.h5')
+    encoder.load_weights( "models/encoder.h5"   )
+    decoder_A.load_weights( "models/decoder_A.h5" )
+    decoder_B.load_weights( "models/decoder_B.h5" )
 except:
     pass
 
-
 def save_model_weights():
-    encoder.save_weights('models/encoder.h5')
-    decoder_A.save_weights('models/decoder_A.h5')
-    decoder_B.save_weights('models/decoder_B.h5')
-    print('save model weights')
-
-
-def show_sample(test_A, test_B):
-    figure_A = numpy.stack([
-        test_A,
-        autoencoder_A.predict(test_A),
-        autoencoder_B.predict(test_A),
-        ], axis=1)
-    figure_B = numpy.stack([
-        test_B,
-        autoencoder_B.predict(test_B),
-        autoencoder_A.predict(test_B),
-        ], axis=1)
-
-    figure = numpy.concatenate([figure_A, figure_B], axis=0)
-    figure = figure.reshape((4, 7) + figure.shape[1:])
-    figure = stack_images(figure)
+    encoder.save_weights( "models/encoder.h5"   )
+    decoder_A.save_weights( "models/decoder_A.h5" )
+    decoder_B.save_weights( "models/decoder_B.h5" )
+    print( "save model weights" )
 
-    figure = numpy.clip(figure * 255, 0, 255).astype('uint8')
+images_A = get_image_paths( imageAFolder )
+images_B = get_image_paths( imageBFolder )
+images_A = load_images( images_A ) / 255.0
+images_B = load_images( images_B ) / 255.0
 
-    cv2.imwrite('_sample.jpg', figure)

```

Comment:
Removing this line will remove any sort of preview for those who are using the Docker version.